x-pack/plugins/security_solution/public/management/components/policy_response/policy_response.tsx

```diff
@@ -143,42 +59,156 @@ export const PolicyResponse = memo(
     policyResponseActions,
     policyResponseAttentionCount,
   }: PolicyResponseProps) => {
-    const generateId = useMemo(() => htmlIdGenerator(), []);
-    return (
-      <>
-        {Object.entries(policyResponseConfig).map(([key, val]) => {
+    const getEntryIcon = useCallback(
+      (status: HostPolicyResponseActionStatus, unsuccessCounts: number) =>
+        status === HostPolicyResponseActionStatus.success ? (
+          <EuiHealth
+            color="success"
+            data-test-subj="endpointPolicyResponseStatusSuccessHealth"
+            className="policyResponseStatusHealth"
+          />
+        ) : status === HostPolicyResponseActionStatus.unsupported ? (
+          <EuiHealth
+            color="subdued"
+            data-test-subj="endpointPolicyResponseStatusSuccessHealth"
+            className="policyResponseStatusHealth"
+          />
+        ) : (
+          <EuiNotificationBadge data-test-subj="endpointPolicyResponseStatusAttentionHealth">
+            {unsuccessCounts}
+          </EuiNotificationBadge>
+        ),
+      []
+    );
+
+    const getConcernedActions = useCallback(
+      (concernedActions: ImmutableArray<string>) => {
+        return concernedActions.map((actionKey) => {
+          const action = policyResponseActions.find(
+            (currentAction) => currentAction.name === actionKey
+          ) as ImmutableObject<HostPolicyResponseAppliedAction>;
+
+          return {
+            label: (
+              <EuiText
+                color={
+                  action.status !== HostPolicyResponseActionStatus.success &&
+                  action.status !== HostPolicyResponseActionStatus.unsupported
+                    ? 'danger'
+                    : 'default'
+                }
+                data-test-subj="endpointPolicyResponseAction"
+              >
+                {formatResponse(actionKey)}
+              </EuiText>
+            ),
+            id: actionKey,
+            className:
+              action.status !== HostPolicyResponseActionStatus.success &&
+              action.status !== HostPolicyResponseActionStatus.unsupported
+                ? 'policy-response-action-expanded'
+                : '',
+            icon: getEntryIcon(
+              action.status,
+              action.status !== HostPolicyResponseActionStatus.success ? 1 : 0
+            ),
+            children: [
+              {
+                label: (
+                  <PolicyResponseActionItem
+                    status={action.status}
+                    actionTitle={action.name}
+                    actionMessage={action.message}
+                    // actionButtonLabel="Do something" // TODO
+                    // actionButtonOnClick={() => {}} // TODO
```

Comment:
??

Reply:
This is going to be used in future pr's but is useless at this point. I can remove those comments if can create any confusion

Reply:
fine to leave it in